incubator/binding-kafka-grpc/src/test/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/config/KafkaGrpcConditionConfigAdapterTest.java

```diff
@@ -51,6 +51,7 @@ public void shouldReadCondition()
                 "    \"topic\": \"responses\",\n" +
                 "    \"reply-to\": \"requests\",\n" +
                 "    \"key\": \"test\",\n" +
+                "    \"method\": \"test/*\",\n" +
```

Comment:
Suggest we move this after `headers`, so `key` and `headers` filter are consecutive.